parts/kubernetesmasteraddons-heapster-deployment.yaml

```diff
@@ -40,11 +40,9 @@ spec:
         - "--extra-cpu=0.5m"
         - "--memory=140Mi"
         - "--extra-memory=4Mi"
-        - "--threshold=5"
```

Comment:
It seems it has rewritten entirely and references changed from 1.7 to 2.0 everywhere https://github.com/kubernetes/autoscaler/commit/93942efb7530a7d266dd7d7a0c83b0417b049471